src/ddmd/astnull.d

```diff
@@ -38,47 +38,47 @@ struct ASTNull
     alias PROTexport        = PROTKIND.PROTexport;
     alias PROTundefined     = PROTKIND.PROTundefined;
 
-    static int MODconst            = 0;
```

Comment:
Make it `static immutable int` and you get the best of both worlds. It's an lvalue which is known at CT.